### ISSUE TITLE
Weighted Temporal Edges

### ISSUE THREAD

**Issue by wolfiex** [uzhdag/pathpy#46]
There isn't an option to assign edge weights when adding new links to a temporal network.

**Comment by IngoScholtes** [comment]
Indeed, time-stamped edges currently don't have weights as it is not clear how the weight semantics should be included in the temporal path calculation. 

But we'll add an option for weighted edges, while ignoring them in the path calculation.

**Comment by wolfiex** [comment]
How does the code currently deal with multiple identical edges of the same timestamp?

**Comment by IngoScholtes** [comment]
Right now, multiple identical edges with the same time stamp will be stored as separate objects. If you create an aggregate static network from it, those multiple edges will be reflected in the edge weights.

See e.g. in this example: 

```
import pathpy as pp
t = pp.TemporalNetwork()
t.add_edge('a', 'b', 1)
t.add_edge('a', 'b', 1)
n = pp.Network.from_temporal_network(t)
print(n.edges[('a', 'b')])
```

This gives the output: `{'weight': 2.0}`
